mautic/mautic
# Improve no campaings lead modal message.

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ 5.x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

See issue https://github.com/mautic/mautic/issues/13472
This PR tries to improve that, by making a slight change to the header title of the modal, and to add a no-campaigns found message. 

When there is a campaign. 
![image](https://github.com/mautic/mautic/assets/1926222/08217640-fa6a-46b6-a6be-3e94e3d44dc6)

When there is no campaign. 
![image](https://github.com/mautic/mautic/assets/1926222/14ad75f7-cf9c-4288-91d7-101cc88e25ee)




#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Check campaign modal for contact in campaign. 
3. Check campaign modal for contact not in campaign.
